notebook/confidence_intervals.ipynb

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Not sure why this notebook was added to the PR, was this intentional?

Reply:
Sorry, that was supposed to be added in another PR. I will remove it.